www/student.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+// Load data from URL
+function loadDefaultData() {
+  if (location.hash !== '') {
+    const urlList = JSON.parse(atob(location.hash.split('#')[1]))
+
+    document.querySelector('#code-area').value = urlList[0]
+    document.querySelector('#output-text').value = urlList[1]
+  }
+}
+loadDefaultData()
+
+// Make check output button function
+document.querySelector('#run').addEventListener('click', function () {
+  const codeText = document.querySelector('#code-area').value
+  const outputText = document.querySelector('#output-text').value
+  const label = document.querySelector('#checkCodeResult')
```

Comment:
checkCodeResult should use a hyphen like the other ids. Style guide:

https://google.github.io/styleguide/htmlcssguide.html#id_Attributes